### PR TITLE
[AIRFLOW-6381] Remove styling based on DAG id from DAGs page

### PR DESCRIPTION
**Pull request by tigerjz32**
Issue link: [AIRFLOW-6381](https://issues.apache.org/jira/browse/AIRFLOW-6381/)

- [x] Description above provides context of the change
- [x] Commit message contains [\[AIRFLOW-XXXX\]](https://issues.apache.org/jira/browse/AIRFLOW-XXXX) or `[AIRFLOW-XXXX]` for document-only changes
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by mik-laj** [comment]
Something is wrong with this change. It contains a huge amount of commit. Can you fix it?

**Comment by kaxil** [comment]
I have updated the base so should be fine now.

File: airflow/www/templates/airflow/dags.html
Comment (kaxil) [inline airflow/www/templates/airflow/dags.html:91]: @mik-laj Do you know if we have custom CSS somewhere based on DAG ID

**Review by kaxil — APPROVED** [review]
@tigerjz32 Thanks for raising this PR

**Comment by kaxil** [comment]
Waiting for the CI to pass before I merge

File: airflow/www/templates/airflow/dags.html
Comment (kaxil) [reply airflow/www/templates/airflow/dags.html:91]: Don't think we do as dag_id can be literally anything!

**Comment by tigerjz32** [comment]
Thank you @kaxil!

File: airflow/www/templates/airflow/dags.html
Comment (mik-laj) [reply airflow/www/templates/airflow/dags.html:91]: I've seen it before, but I believed it had a purpose. :-D

**Pull request merged**
